substrate/client/network/benches/notifications_protocol.rs

```diff
@@ -36,19 +36,16 @@ use std::{sync::Arc, time::Duration};
 use substrate_test_runtime_client::runtime;
 use tokio::{sync::Mutex, task::JoinHandle};
 
-const SMALL_PAYLOAD: &[(u32, usize, &'static str)] = &[
-	// (Exponent of size, number of notifications, label)
-	(6, 100, "64B"),
-	(9, 100, "512B"),
-	(12, 100, "4KB"),
-	(15, 100, "64KB"),
-];
-const LARGE_PAYLOAD: &[(u32, usize, &'static str)] = &[
-	// (Exponent of size, number of notifications, label)
-	(18, 10, "256KB"),
-	(21, 10, "2MB"),
-	(24, 10, "16MB"),
-	(27, 10, "128MB"),
+const NUMBER_OF_NOTIFICATIONS: usize = 10;
```

Comment:
10 notifications are likely not enough to reach the full TCP speed. Can we include the "connection pre-heating" phase where we just transfer big notifications from one peer to another and don't measure anything? The idea is to let TCP congestion window scale to be bigger than one notification size, so waiting for TCP ACKs doesn't slow down the connection.

I would say we need to pump 32MB through the connection for cwnd to reach 16 MB. Might be it won't actually reach 16MB due to TCP stack limits, but at least this is the best we can do.

Can we add such pre-heating before we send any actual notifications/requests we measure?

Reply:
How many notifications do we need to preheat? I suggest using only 10 notifications because there is no difference in results between 10 and 100 notifications.